thoth/storages/__init__.py

```diff
@@ -41,4 +43,4 @@
 
 
 __name__ = "thoth-storages"
-__version__ = "0.14.4"
+__version__ = "0.14.5"
```

Comment:
Versions are automatically managed by Kebechet, please do not change them manually.

Reply:
okay, cool I didn't know that. :)